server/channels/store/sqlstore/group_store.go

```diff
@@ -1284,34 +1284,42 @@ func (s *SqlGroupStore) ChannelMembersToRemove(channelID *string) ([]*model.Chan
 }
 
 func (s *SqlGroupStore) groupsBySyncableBaseQuery(st model.GroupSyncableType, t selectType, syncableID string, opts model.GroupSearchOpts) sq.SelectBuilder {
-	selectStrs := map[selectType]string{
-		selectGroups:      "UserGroups.*, gs.SchemeAdmin AS SyncableSchemeAdmin",
-		selectCountGroups: "COUNT(*)",
+	var query sq.SelectBuilder
```

Comment:
We start with the `SelectBuilder` at the outset, instead of a convoluted `fmt.Sprintf` later on.